src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -86,36 +86,46 @@ const ioTask = ({ name }) => _.nth(-2, name.split('.'))
 const ioVariable = ({ name }) => _.nth(-1, name.split('.'))
 const ioType = ({ inputType, outputType }) => (inputType || outputType).match(/(.*?)\??$/)[1] // unify, and strip off trailing '?'
 
+
 const WorkflowIOTable = ({ which, inputsOutputs: data, config, errors, onChange, onSetDefaults, onBrowse, suggestions, readOnly }) => {
+  const [sort, setSort] = useState({ field: 'taskVariable', direction: 'asc' })
+
+  const sortedData = _.orderBy([function(o) {
+    return Utils.switchCase(sort.field,
+      ['taskVariable', () => ioTask(o).toLowerCase()],
+      ['workflowVariable', () => ioVariable(o).toLowerCase()]
+    )
+  }, 'optional'], [sort.direction, 'asc'], data)
+
   return h(AutoSizer, [
     ({ width, height }) => {
       return h(FlexTable, {
         width, height,
-        rowCount: data.length,
+        rowCount: sortedData.length,
         columns: [
           {
             size: { basis: 350, grow: 0 },
-            headerRenderer: () => h(HeaderCell, ['Task name']),
+            headerRenderer: () => h(Sortable, { sort, field: 'taskVariable', onSort: setSort }, ['Task name']),
```

Comment:
The `HeaderCell` adds some styling that we'd like to keep. They should compose fine, so we can continue to render the `HeaderCell` inside of the `Sortable`.

Reply:
That makes sense, though not sure how to implement this. Could you point me to a similar example in the code base?

Reply:
```suggestion
            headerRenderer: () => h(Sortable, { sort, field: 'taskVariable', onSort: setSort }, [h(HeaderCell, ['Task name'])]),
```

Reply:
See Isaac's suggestion, and to explain it: Because `Sortable` is a well-behaved composable container, its `children` can have arbitrary content. So instead of just the string, we can render an element there, in this case the `HeaderCell`.